app/src/main/java/org/coepi/android/ui/onboarding/OnboardingViewModel.kt

```diff
@@ -14,10 +27,33 @@ class OnboardingViewModel(
     private val preferences: Preferences
 ) : ViewModel() {
 
+    val context: Context = TODO("Need this to pick up main context, not sure how in VM");
+    val activity: Activity = TODO("Need this to attach to main activity, not sure how in VM");
```

Comment:
ViewModel shouldn't be coupled with UI. You have to add the permission via `BlePreconditions`